packages/bot-web-ui/src/stores/flyout-help-store.js

```diff
@@ -77,6 +78,7 @@ export default class FlyoutHelpStore {
         if (block_node) {
             const target_blocks = this.xml_list_group[block_node];
             const block_type = target_blocks[0].getAttribute('type');
+            /* eslint-disable-next-line import/namespace */
```

Comment:
Added eslint ignore comments, this will be fixed on a separate card:
<img width="1398" alt="Screenshot 2023-06-26 at 2 33 12 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/624823c6-d765-4cf0-81ea-73ebd5ff6c89">

Reply:
for this error adding the below rule to the main package will fix the issue: 

`'import/namespace': ['error', { allowComputed: true }],`

Reply:
cc: @ali-hosseini-deriv

Reply:
I've also pushed a new commit to disable localize-usage

Reply:
shayan i published your changes